Xamarin.Forms.Platform.UAP/VisualElementRenderer.cs

```diff
@@ -297,16 +297,10 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<TElement> e)
 				changed(this, e);
 		}
 
-		protected void UpdateTabStop()
+		protected virtual void UpdateTabStop()
 		{
-			if (_control == null)
-				return;
-			_control.IsTabStop = Element.IsTabStop;
-
-			// update TabStop of children for complex controls (like as DatePicker, TimePicker, SearchBar and Stepper)
```

Comment:
Unfortunately removal of this code may have an impact on 3rd party controls that depend on it. I haven't tried to figure out if thats actually the case but I would like to call out the possibility.

Reply:
This code was added recently in `master` branch and only for `DatePicker`, `TimePicker`, `SearchBar` and `Stepper`. I don't think anyone's noticed him yet. 
If someone on 3rd party controls wants to use it then it will be enough add new interface to 3rd party renderer.